zk/stages/stage_sequence_execute_unwind.go

```diff
@@ -60,7 +63,8 @@ func unwindSequenceExecutionStage(u *stagedsync.UnwindState, s *stagedsync.Stage
 		return err
 	}
 
-	if err = updateSequencerProgress(tx, u.UnwindPoint, fromBatch, 1); err != nil {
+	//TODO: why l1infoindex is 1?
+	if err = updateSequencerProgress(tx, u.UnwindPoint, fromBatch, 1, true); err != nil {
```

Comment:
I don't think we should be setting this to 1 for the info tree index if this is indeed doing that.  It sounds dangerous to go back to 1 as part of this process.  Well highlighted.

Reply:
I think what we should be doing here is going from the unwind point backwards until we find a used L1 info tree index and using this instead.  Then we are actually unwinding back to the real last used index.

Reply:
You are right. "1" here is completely wrong. Currently L1InfoTreeIndexProgress is batch-based. I'm changing it to block-based and then we will have the correct value for this function.